website/docs/r/prometheus_workspace.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+subcategory: "Amazon Managed Service for Prometheus (AMP)"
+layout: "aws"
+page_title: "AWS: aws_prometheus_workspace"
+description: |-
+  Creates a prometheus workspace
+---
+
+# Resource: aws_prometheus_workspace
+
+Provides a managed prometheus workspace.
+
+-> **Note:** only workspace alias can be updated today
+
+## Example Usage
+
+```hcl
+resource "aws_prometheus_workspace" "demo" {
+  alias = "prometheus-test"
+}
+```
+
+## Argument Reference
+
+The following argument is supported:
+
+* `alias` - (Optional) The alias of the prometheus workspace. See more [in AWS Docs](https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-onboard-create-workspace.html).
+
+## Attributes Reference
```

Comment:
Might want to also output the ARN, workspace ID. Also add the `endpoint_remote_write_url` and `endpoint_query_url` could also help the end user by not having to reconstruct these url from the `prometheus_endpoint`.

Reply:
the workspace id is the terraform resource id. I missed the ARN in the documentation, so nice reminder.
Also I'll add the `remote_write` and `query` endpoints